setup.py

```diff
@@ -29,19 +37,349 @@ def run(self):
 
         egg_info.run(self)
 
-if __name__ == "__main__":
-    install_requires = []
-    if os.environ.get("GK_BUILD_WHEELS", None) is not None:
-        install_requires = [
-            "appdirs>=1.4.0",
-            "numpy<2.0dev0",
-            "google-cloud-storage>=2.10.0",
-            "boto3",
-            "tqdm",
-            "setuptools",
-            "importlib-metadata",
-            "typing-extensions",
+
+
+libname = "genome_kit"
```

Comment:
Moved with minor modifications from c_ext.py. To view, `git diff main:setup/c_ext.py chore/setuptools:setup.py`

Reply:
I don't think it's a path issue. From my observations, the new versions of setuptools don't load project code at all, only setup.py itself. I could be wrong, I can try this style of import as well.